sklearn/model_selection/tests/test_validation.py

```diff
@@ -1421,3 +1421,31 @@ def test_permutation_test_score_pandas():
         check_series = lambda x: isinstance(x, TargetType)
         clf = CheckingClassifier(check_X=check_df, check_y=check_series)
         permutation_test_score(clf, X_df, y_ser)
+
+
+def test_fit_and_score():
+    from sklearn.tests.test_grid_search import FailingClassifier
+    from sklearn.model_selection._validation import _fit_and_score
```

Comment:
While I know it's a bad practice to import the protected method _fit_and_score, the required warning message would be raised only by passing the error_score parameter which I couldn't do with `cross_val_score` or `cross_validate` public functions without adding an additional parameter (and I don't know if it'd be great to add a  parameter for this trivial purpose)

Reply:
Well `GridSearchCV(est,{},refit=False).fit(X,y)` is basically the same as `cross_val_score`...

But I think this is perfectly fine in unit testing. Still not sure why you need to import it locally, not at the top of the file.

Reply:
@jnothman I've fixed it. But the Travis CI build fails with a weird error. What do you suggest?